uui-core/src/data/lenses/lensesImpl.tsx

```diff
@@ -58,6 +59,33 @@ export function prop<TObject, TKey extends keyof TObject>(name: TKey): ILensImpl
     };
 }
 
+export function get<TItem, TId>(id: TId): ILensImpl<ItemsMap<TId, TItem>, TItem> {
```

Comment:
Let's:
- generalize this Lens to support any Map-like object. We can re-use the trick we used to abstract Map in Tree via IMap interface. This approach allows to support a native Map, as well as other Map-like objects. Including immutable Maps like I.Map - they can implement constructor as no-op.
- we need a better name for both ILensImpl and LensBuilder method. I suggest ```mapItem```, but let's discuss - probably there'll be a better ideas.